### PR TITLE
Some quick housekeeping of README/examples

### PR DESCRIPTION
**Pull request by cgmckeever**
Clarify some README/example items that didnt track based on previous PR suggestions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ void loop()
 
 ```
 server->on("/settings.html", HTTPMethod::HTTP_GET, [server](){
-    configManager.streamFile(settingsHTML, mimeHTML);
+    configManager.streamFile(configHTMLFile, mimeHTML);
```
Comment (cgmckeever) [inline README.md]: whoops

File: README.md [hunk 2]
```diff
@@ -203,15 +203,15 @@ server->on("/settings.html", HTTPMethod::HTTP_GET, [server](){
 
 ## GET /
 
-###### *(AP,API)*
+###### Modes: *AP and API*
```
Comment (cgmckeever) [inline README.md]: looked a bit awkward

File: README.md [hunk 3]
```diff
@@ -232,23 +232,25 @@ ssid=access point&password=some password
...
++ Will print to Serial Monitor is `DEBUG_MODE = true`
+
 + Response 200 *(application/json)*
 
 ```json
 {
   "ssid": "access point name",
   "strength": *int*, 
-  "security": "none" *or* "enabled"
+  "security": *bool*
```
Comment (cgmckeever) [inline README.md]: whoops

File: examples/simple/simple.ino [hunk 4]
```diff
@@ -20,6 +20,7 @@ void createCustomRoute(WebServer *server) {
 }
 
 void setup() {
+    DEBUG_MODE = true; // will log to serial monitor
```
Comment (cgmckeever) [inline examples/simple/simple.ino]: figured for the newcomer, having it PRINT was a good move

File: README.md [hunk 5]
```diff
@@ -27,6 +27,24 @@ Include the library in your sketch
...
+    char password[20];
+} config;
+
+struct Metadata {
+    int8_t version;
+} meta;
+
+```
+
+> Note: These are examples of possible settings. The `config` is required, these values are arbitrary.
```
Comment (cgmckeever) [inline README.md]: `password` `hour` `enabled` tripped me up in regards to usage versus arbitrary custom settings

File: README.md [hunk 6]
```diff
@@ -52,6 +70,14 @@ In your loop function, run the manager loop
 configManager.loop();
 ```
 
+To access your config data within your loop/device code
+
+```cpp
+String name = config.name;
+```
+
```
Comment (cgmckeever) [inline README.md]: This was completely obvious, I fought myself trying to do this with bad code .. but it was simple -- just putting this up front so others know

File: README.md [hunk 7]
```diff
@@ -52,6 +70,23 @@ In your loop function, run the manager loop
 configManager.loop();
 ```
 
+To access your config data through device code
+
+```cpp
+String name = config.name;
```
Comment (nrwiersma) [inline README.md]: Isnt this a type mismatch?

Comment (cgmckeever) [reply README.md]: probably - updated

**Pull request merged**